tools/c7n_azure/c7n_azure/query.py

```diff
@@ -42,9 +42,17 @@ def filter(self, resource_manager, **params):
         if extra_args:
             params.update(extra_args)
 
-        op = getattr(getattr(resource_manager.get_client(), enum_op), list_op)
-        data = [r.serialize(True) for r in op(**params)]
+        params.update(m.extra_args(resource_manager))
 
+        op = getattr(getattr(resource_manager.get_client(), enum_op), list_op)
+        result = op(**params)
+
+        try:
+            data = [r.serialize(True) for r in result]
+        except TypeError:
+            # This is due to an issue with inconsistency in Azure SDK:
+            # https://github.com/Azure/azure-sdk-for-python/issues/6941
+            data = [r.serialize(True) for r in result.value]
```

Comment:
Do we want to handle all the edge cases here or just keep this clean and override in resources?  This will leave a stack trace about `value` not being a valid field for any failure now.